opentelekomcloud/acceptance/dds/data_source_opentelekomcloud_dds_instance_v3_test.go

```diff
@@ -57,9 +60,9 @@ resource "opentelekomcloud_dds_instance_v3" "instance_1" {
     version        = "3.4"
     storage_engine = "wiredTiger"
   }
-  vpc_id            = "%s"
-  subnet_id         = "%s"
-  security_group_id = opentelekomcloud_networking_secgroup_v2.sg_acc.id
+  vpc_id            = data.opentelekomcloud_vpc_subnet_v1.shared_subnet.vpc_id
+  subnet_id         = data.opentelekomcloud_vpc_subnet_v1.shared_subnet.network_id
```

Comment:
It's better to use `data.opentelekomcloud_vpc_subnet_v1.shared_subnet.id`, as field is `subnet_id`